src/web/components/SignInGate/gateDesigns/shared.tsx

```diff
@@ -9,17 +9,6 @@ export const signInGateContainer = css`
 	${from.desktop} {
 		min-height: 600px;
 	}
-
-	/* This needs to be here because link styles are applied globally to the article body :/ */
-	a {
-		text-decoration: underline;
-		border-bottom: none;
-		color: ${palette.text.primary};
-
-		:hover {
-			border-bottom: none;
-		}
-	}
```

Comment:
we no longer need this as we can apply `data-ignore="global-link-styling"` to any link that shouldn't be globally styled